File: 백준/Bronze/25304. 영수증/영수증.cc
#include <iostream>
using namespace std;

int main(){
    
 int x, n, sum, a, b;
 cin >> x;
 cin >> n;
 sum = 0;
 
 for (int i = 0; i < n; i++){
    cin >> a >> b;
    sum += (a * b);
 }
 
 if (sum == x) 
    cout << "Yes" << endl;
 else
    cout << "No" << endl;
 
 return 0;   
}

File: 백준/Bronze/25304. 영수증/영수증.py
x = int(input(""))
n = int(input(""))
sum = 0
for i in range(n):
    tup = input("")
    a, b = tup.split(" ")
    sum += int(a) * int(b)
if x == sum:
    print("Yes")
else:
    print("No")